ERROR [2021-01-10T15:23:22,048] : [http-nio-8080-exec-2] : ResponseExceptionHandler: manejarTodasExcepciones(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=INVALID_CREDENTIALS, code=SOAINT999, backendMesage=Error generico de la aplicación., fecha=2021-01-10T15:23:22.047]
ERROR [2021-01-10T15:23:33,251] : [http-nio-8080-exec-4] : ResponseExceptionHandler: manejarTodasExcepciones(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=INVALID_CREDENTIALS, code=SOAINT999, backendMesage=Error generico de la aplicación., fecha=2021-01-10T15:23:33.251]
ERROR [2021-01-10T15:28:47,993] : [http-nio-8080-exec-3] : ResponseExceptionHandler: manejarTodasExcepciones(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=INVALID_CREDENTIALS, code=SOAINT999, backendMesage=Error generico de la aplicación., fecha=2021-01-10T15:28:47.992]
ERROR [2021-01-10T15:28:55,854] : [http-nio-8080-exec-5] : ResponseExceptionHandler: manejarTodasExcepciones(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=INVALID_CREDENTIALS, code=SOAINT999, backendMesage=Error generico de la aplicación., fecha=2021-01-10T15:28:55.854]
ERROR [2021-01-10T15:39:33,338] : [http-nio-8080-exec-2] : ResponseExceptionHandler: manejarTodasExcepciones(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=INVALID_CREDENTIALS, code=SOAINT999, backendMesage=Error generico de la aplicación., fecha=2021-01-10T15:39:33.337]
ERROR [2021-01-10T15:40:01,343] : [http-nio-8080-exec-4] : ResponseExceptionHandler: manejarTodasExcepciones(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=INVALID_CREDENTIALS, code=SOAINT999, backendMesage=Error generico de la aplicación., fecha=2021-01-10T15:40:01.342]
ERROR [2021-01-10T15:40:05,041] : [http-nio-8080-exec-6] : ResponseExceptionHandler: manejarTodasExcepciones(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=INVALID_CREDENTIALS, code=SOAINT999, backendMesage=Error generico de la aplicación., fecha=2021-01-10T15:40:05.041]
ERROR [2021-01-10T15:41:40,864] : [http-nio-8080-exec-3] : ResponseExceptionHandler: manejarTodasExcepciones(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=INVALID_CREDENTIALS, code=SOAINT999, backendMesage=Error generico de la aplicación., fecha=2021-01-10T15:41:40.864]
ERROR [2021-01-10T15:44:42,322] : [http-nio-8080-exec-2] : ResponseExceptionHandler: manejarTodasExcepciones(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=INVALID_CREDENTIALS, code=SOAINT999, backendMesage=Error generico de la aplicación., fecha=2021-01-10T15:44:42.322]
ERROR [2021-01-10T15:44:49,433] : [http-nio-8080-exec-4] : ResponseExceptionHandler: manejarTodasExcepciones(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=INVALID_CREDENTIALS, code=SOAINT999, backendMesage=Error generico de la aplicación., fecha=2021-01-10T15:44:49.432]
ERROR [2021-01-10T15:48:54,603] : [http-nio-8080-exec-3] : ResponseExceptionHandler: manejarTodasExcepciones(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=INVALID_CREDENTIALS, code=SOAINT999, backendMesage=Error generico de la aplicación., fecha=2021-01-10T15:48:54.603]
ERROR [2021-01-10T18:41:21,165] : [http-nio-8080-exec-10] : ResponseExceptionHandler: MethodArgumentTypeMismatchException(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=Failed to convert value of type 'java.lang.String' to required type 'java.lang.Integer'; nested exception is java.lang.NumberFormatException: For input string: "AKSJDHASKD", code=SOAINT002, backendMesage=Error en los argumentos(datos, parametros)., fecha=2021-01-10T18:41:21.165]
ERROR [2021-01-10T18:44:03,961] : [http-nio-8080-exec-4] : ResponseExceptionHandler: handleMethodArgumentNotValid(): com.soaint.exception.ExceptionResponse@5360295c
ERROR [2021-01-10T18:45:54,764] : [http-nio-8080-exec-1] : ResponseExceptionHandler: manejarModeloException(): ExceptionResponseCustom [httpStatus=404 NOT_FOUND, message=ID NO ENCONTRADO 1121, code=SOAINT001, backendMesage=Recurso no encontrado., fecha=2021-01-10T18:45:54.764]
ERROR [2021-01-10T18:46:08,412] : [http-nio-8080-exec-2] : ResponseExceptionHandler: MethodArgumentTypeMismatchException(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=Failed to convert value of type 'java.lang.String' to required type 'java.lang.Integer'; nested exception is java.lang.NumberFormatException: For input string: "sadkj", code=SOAINT002, backendMesage=Error en los argumentos(datos, parametros)., fecha=2021-01-10T18:46:08.412]
ERROR [2021-01-10T18:53:17,896] : [http-nio-8080-exec-7] : ResponseExceptionHandler: runtimeException(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=Unable to find com.soaint.entity.Cliente with id 1; nested exception is javax.persistence.EntityNotFoundException: Unable to find com.soaint.entity.Cliente with id 1, code=, backendMesage=, fecha=2021-01-10T18:53:17.896]
ERROR [2021-01-10T18:53:25,908] : [http-nio-8080-exec-8] : ResponseExceptionHandler: runtimeException(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=could not execute statement; SQL [n/a]; constraint [null]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement, code=, backendMesage=, fecha=2021-01-10T18:53:25.908]
ERROR [2021-01-10T18:53:43,343] : [http-nio-8080-exec-10] : ResponseExceptionHandler: runtimeException(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=Unable to find com.soaint.entity.Cliente with id 3; nested exception is javax.persistence.EntityNotFoundException: Unable to find com.soaint.entity.Cliente with id 3, code=, backendMesage=, fecha=2021-01-10T18:53:43.343]
ERROR [2021-01-10T18:54:22,724] : [http-nio-8080-exec-3] : ResponseExceptionHandler: runtimeException(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=could not execute statement; SQL [n/a]; constraint [null]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement, code=, backendMesage=, fecha=2021-01-10T18:54:22.724]
ERROR [2021-01-10T18:54:38,800] : [http-nio-8080-exec-4] : ResponseExceptionHandler: runtimeException(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=could not execute statement; SQL [n/a]; constraint [null]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement, code=, backendMesage=, fecha=2021-01-10T18:54:38.799]
ERROR [2021-01-10T18:55:42,405] : [http-nio-8080-exec-2] : ResponseExceptionHandler: runtimeException(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=could not execute statement; SQL [n/a]; constraint [null]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement, code=, backendMesage=, fecha=2021-01-10T18:55:42.405]
ERROR [2021-01-10T19:04:17,792] : [http-nio-8080-exec-9] : ResponseExceptionHandler: runtimeException(): ExceptionResponseCustom [httpStatus=500 INTERNAL_SERVER_ERROR, message=could not execute statement; SQL [n/a]; constraint [null]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement, code=, backendMesage=, fecha=2021-01-10T19:04:17.792]
